content/docs/user-guide/assets/asset-processor/debugging.md

```diff
@@ -122,3 +122,107 @@ If you're a game artist and you're having issues running Asset Processor, the is
+
+### Effect
+
+The frustration can lead to the scene author to tweak minor data in the original source scene (i.e. the Blender file) and re-exporting to attempt to resolve strange errors from the O3DE scene pipeline. The scene pipeline does many processing steps so it can be confusing to determine which the scene node data (e.g. Transform Data, Mesh Data) were discovered and the rules used to import the scene nodes. The scene pipeline events can be overridden by either Python scripts or C++ code to modify the scene manifest rules. This can lead to confusion of what rules were used to generate the product assets.
+
+### Example
+
+As source scene assets become more complex, teams will eventually need to debug strange output from the scene pipeline.
+
+The team may encounter:
+ - Render models not aligning with collision meshes
+ - Materials end up with unexpected settings or textures
+ - Finding extra models in the scene
+ - User defined properties not showing up with the correct values
+ - Unexpected groupings of meshe nodes stored in a render model
+
+### Common Causes
+
+#### AssImp issues
+
+The scene pipeline uses the AssImp library to import source scene files into a scene graph. The scene graph is the in-memory representation of the source scene file in the pipeline. It is possible that the source scene file looks different in the scene graph due to how the AssImp library imports the file.
+
+#### Missing user defined properties
+The user defined properties that can be stored in a source scene file can be imported with unexpected results such as missing keys or changed values. There could be a mismatch in what the AssImp library will import, an option to export custom properties might have been missed, or the scene pipeline might now support the exact value types from the source scene file.
+
+For more information, refer to [Scene API: User Defined Properties](docs/user-guide/assets/pipeline/scene-api-udp/).
+
+#### Wrong scene manifest rules used
+A technical content creator (such as a Technical Artist) that is authoring or debugging a script might find some unexpected results for some source scene assets. The Python scripts output print() commands in the asset’s log files, but this may not be enough to determine what the script is affecting. The “debug output” flag is another good way to determine what is happening in the affected scripted pipeline.
+
+### Solutions
+
+#### Enable the Debug Output Feature
+The “debug output” flag is a good feature/flag to use to see what the scene pipeline produced for the scene graph and scene manifest. The scene graph is considered as immutable after the source scene is imported from the AssImp library. The scene manifest can be updated during the scene pipeline events. 

```

Comment:
nit: Use "feature flag" at minimum. Do O3DE devs know what we mean by feature flags.

```suggestion
The “debug output” flag is a feature flag that can be used to see what the scene pipeline produced for the scene graph and scene manifest. The scene graph is considered immutable after the source scene is imported from the AssImp library. The scene manifest can be updated during the scene pipeline events. 

```